fe1-web/src/features/wallet/objects/Token.ts

```diff
@@ -68,10 +68,16 @@ export const getCurrentPopTokenFromStore =
       throw new Error('Cannot retrieve pop token: roll call id is undefined');
     }
 
-    const rollCall = getEventById(rollCallId) as WalletFeature.RollCall;
+    const rollCall = getRollCallById(rollCallId);
+    if (!rollCall) {
+      throw new Error(`Cannot retrieve pop token: did not find a roll call with id ${rollCallId}`);
+    }
+
     const token = await generateToken(lao.id, rollCallId);
-    if (rollCall.containsToken(token)) {
-      return token;
+
+    if (!rollCall.containsToken(token)) {
+      throw new Error('Cannot retrieve pop token: roll call does not contain this token');
```

Comment:
outside the scope of this PR: typed errors would be nice here, to distinguish between line 73, a "this-should-never-happen-and-all-our-expectations-have-been-violated" error, and line 79, a "tough-luck-you-have-no-token" kind of error

Reply:
Yes I agree!